packages/meditrak-app/app/entityMenu/EntityList.js

```diff
@@ -188,7 +182,7 @@ export class EntityList extends PureComponent {
     const { isOpen, numberToShow } = this.state;
 
     // if base query is empty, the survey is probably misconfigured
```

Comment:
Change 'query' to 'fetch' ?
` // if base fetch is empty, the survey is probably misconfigured`